tests/smoke_test.py

```diff
@@ -0,0 +1,4 @@
+import pytest
```

Comment:
The import of pytest here is not strictly needed.

Reply:
Ah good to know! Thanks!